pydomcfg/tests/bathymetry.py

```diff
@@ -147,29 +147,34 @@ def _calc_rmax(depth):
     Parameters
     ----------
     depth: float
-            Bottom depth (units: m).
+        Bottom depth (units: m).
 
     Returns
     -------
     rmax: float
-            Slope steepness value (units: None)
+        Slope steepness value (units: None)
     """
-    depth = depth.reset_index(list(depth.dims))
+
+    # Replace land with zeros
+    depth = depth.where(depth > 0, 0)
 
     both_rmax = []
     for dim in depth.dims:
 
-        # (H[0] - H[1]) / (H[0] + H[1])
-        depth_diff = depth.diff(dim)
-        depth_rolling_sum = depth.rolling({dim: 2}).sum().dropna(dim)
-        rmax = depth_diff / depth_rolling_sum
+        rolled = depth.rolling({dim: 2}).construct("tmp_dim")
+
+        # |(H[0] - H[1])| / (H[0] + H[1])
+        # First value is NaN
+        diff = rolled.diff("tmp_dim").squeeze("tmp_dim")
+        rmax = np.abs(diff) / rolled.sum("tmp_dim")
 
-        # (R[0] + R[1]) / 2
-        rmax = rmax.rolling({dim: 2}).mean().dropna(dim)
+        # (rmax[0] + rmax[1]) / 2
+        # First two values are NaN
+        rmax = rmax.rolling({dim: 2}).mean(skipna=True)
```

Comment:
I actually need to construct here as well... Looks like `skipna=True` has not effect otherwise...